rpc/subscription.go

```diff
@@ -181,8 +176,11 @@ func (n *Notifier) activate() error {
 	return nil
 }
 
-func (n *Notifier) send(sub *Subscription, data json.RawMessage) error {
-	params, _ := json.Marshal(&subscriptionResult{ID: string(sub.ID), Result: data})
+func (n *Notifier) send(sub *Subscription, data interface{}) error {
+	params, _ := json.Marshal(struct {
+		ID     string      `json:"subscription"`
+		Result interface{} `json:"result,omitempty"`
+	}{ID: string(sub.ID), Result: data})
```

Comment:
Why not just modify `subscriptionResult`?

Reply:
`subscriptionResult` this struct is also used by the websocket client, modify that struct may affect the client's marshal/unmarshal logic. Let me see if we can change it together